.github/workflows/vkd3d-arm64ec.yml

```
@@ -168,350 +168,50 @@ jobs:
+          wget -q https://github.com/KhronosGroup/glslang/releases/download/main-tot/glslang-main-linux-Release.zip
+          unzip -q glslang-main-linux-Release.zip -d glslang
+          rm glslang-main-linux-Release.zip
+
+      - name: Add GLSLang to PATH
+        run: |
+          echo "$PWD/glslang/bin" >> $GITHUB_PATH
+          echo "GLSLang version:"
+          $PWD/glslang/bin/glslangValidator --version 2>/dev/null || $PWD/glslang/bin/glslang --version 2>/dev/null || true
+
       - name: Clone Source
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:4:48: Double quote to prevent globbing and word splitting

(shellcheck)